### PR TITLE
Rename EOD to Historical in code and files

### PR DESCRIPTION
**Pull request by the-praxs**
Windows PowerToys FTW

### CODE REVIEW

File: openbb_sdk/extensions/charting/openbb_charting/core/plotly.html [hunk 1]
```diff
@@ -4083,7 +4083,7 @@
...
...AAAYAAGAA4AHAAqADoATABcAAEEbAABAAIEbARdAAAAAAABAAMEbARdBF0AAAACBHIE2AABAAIEbARdAAAAAwYtBHIE2AABAAIEbARdAAAAAQS0AAEAAQRdAAEAAABgAAAAAQACBGwEXQABAAAAYQABAAAAAQAIAAIACgACBG4EtAABAAIEXQRsAAYAAAABAAgAARGsAAIACgA+AAMACAAWACQAAQYtAAEAAgYyBjIAAAAAAAEAAwYyBjIGMgAAAAAAAQACBjIGMgABAAAAYwACAAYAFgACBnQGcwABAAAAAQAAAGMAAQZzAAEAAQYyAAEAAABkAAEAAAABAAgAAhFAAAIGcwWBAAYAAAABAAgAAQB8AAMADAAgAFQAAQAEAAIGdAZzAAEAAAABAAAAZgADAAgAFgAkAAEGLQABAAIGMgYsAAAAAAABAAMGMgYsBiwAAAAAAAEAAgYyBiwAAQAAAGYAAQAEAAEGcwABAAEGLAABAAAAZgABAAAAAQAIAAIADAADBYAGcwZ0AAEAAwYsBi0GMgAGAAAAAQAIAAEAXAADAAwAIAA0AAEABAABBnMAAQABBiwAAQAAAGgAAQAEAAIEtgZzAAEAAAABAAAAaAABAAQAAAABAAIEeQYsAAEAAABoAAEAAAABAAgAAgAMAAMEtgVyBnMAAQADBHkGLAYtAAYAAAABAAgAAQBcAAMADAAgADQAAQAEAAEGcwABAAEGLAABAAAAagABAAQAAgX0BnMAAQAAAAEAAABqAAEABAAAAAEAAgXsBiwAAQAAAGoAAQAAAAEACAACAAwAAwX0BXgGcwABAAMF7AYsBi0ABgAAAAEACAABAFwAAwAMACAANAABAAQAAQZzAAEAAQYsAAEAAABsAAEABAACBnAGcwABAAAAAQAAAGwAAQAEAAAAAQACBiYGLAABAAAAbAABAAAAAQAIAAIADAADBnAFegZzAAEAAwYmBiwGLQAGAAAAAQAIAAEAfAADAAwAIAA0AAEABAABBnMAAQABBiwAAQAAAG4AAQAEAAIEuAZzAAEAAAABAAAAbgADAAgAFgAkAAEGLQABAAIEigYsAAAAAAABAAMEigYsBiwAAAAAAAEAAgSKBiwAAQAAAG4AAQAAAAEACAACAAwAAwS4BYMGcwABAAMEigYsBi0ABgAAAAEACAABAHwAAwAMAEAAVAADA
```
Comment (hjoaquim) [inline openbb_sdk/extensions/charting/openbb_charting/core/plotly.html:4086]: Why is this changed?

File: openbb_sdk/extensions/charting/openbb_charting/core/table.html [hunk 2]
```diff
@@ -296,12 +296,12 @@
...
-`+ s[++c]; switch (p = p.trim(), +h) { case -1: if (p === "BOT") { f[++l] = [], u = 0; continue } else if (p !== "EOD") throw new Error("Unrecognized DIF special command " + p); break; case 0: p === "TRUE" ? f[l][u] = !0 : p === "FALSE" ? f[l][u] = !1 : isNaN(Qr(m)) ? isNaN(Na(m).getDate()) ? f[l][u] = m : f[l][u] = Lt(m) : f[l][u] = Qr(m), ++u; break; case 1: p = p.slice(1, p.length - 1), p = p.replace(/""/g, '"'), p && p.match(/^=".*"$/) && (p = p.slice(2, -1)), f[l][u++] = p !== "" ? p : null; break }if (p === "EOD") break
+`+ s[++c]; switch (p = p.trim(), +h) { case -1: if (p === "BOT") { f[++l] = [], u = 0; continue } else if (p !== "Historical") throw new Error("Unrecognized DIF special command " + p); break; case 0: p === "TRUE" ? f[l][u] = !0 : p === "FALSE" ? f[l][u] = !1 : isNaN(Qr(m)) ? isNaN(Na(m).getDate()) ? f[l][u] = m : f[l][u] = Lt(m) : f[l][u] = Qr(m), ++u; break; case 1: p = p.slice(1, p.length - 1), p = p.replace(/""/g, '"'), p && p.match(/^=".*"$/) && (p = p.slice(2, -1)), f[l][u++] = p !== "" ? p : null; break }if (p === "Historical") break
```
Comment (hjoaquim) [inline openbb_sdk/extensions/charting/openbb_charting/core/table.html:299]: Should this change?

File: openbb_sdk/extensions/charting/openbb_charting/core/table.html [hunk 3]
```diff
@@ -296,12 +296,12 @@
...
+          var u = [], c = Ke(l["!ref"]), f, d = Array.isArray(l); o(u, "TABLE", 0, 1, "sheetjs"), o(u, "VECTORS", 0, c.e.r - c.s.r + 1, ""), o(u, "TUPLES", 0, c.e.c - c.s.c + 1, ""), o(u, "DATA", 0, 0, ""); for (var h = c.s.r; h <= c.e.r; ++h) { a(u, -1, 0, "BOT"); for (var m = c.s.c; m <= c.e.c; ++m) { var p = Ue({ r: h, c: m }); if (f = d ? (l[h] || [])[m] : l[p], !f) { a(u, 1, 0, ""); continue } switch (f.t) { case "n": var g = f.w; !g && f.v != null && (g = f.v), g == null ? f.f && !f.F ? a(u, 1, 0, "=" + f.f) : a(u, 1, 0, "") : a(u, 0, g, "V"); break; case "b": a(u, 0, f.v ? 1 : 0, f.v ? "TRUE" : "FALSE"); break; case "s": a(u, 1, 0, isNaN(f.v) ? f.v : '="' + f.v + '"'); break; case "d": f.w || (f.w = kn(f.z || rt[14], jt(Lt(f.v)))), a(u, 0, f.w, "V"); break; default: a(u, 1, 0, "") } } } a(u, -1, 0, "Historical"); var x = `\r
```
Comment (hjoaquim) [inline openbb_sdk/extensions/charting/openbb_charting/core/table.html:304]: Should this change?

File: openbb_sdk/openbb/package/futures.py [hunk 4]
```diff
@@ -87,107 +86,3 @@ def curve(
             "/futures/curve",
             **inputs,
         )
-
```
Comment (hjoaquim) [inline openbb_sdk/openbb/package/futures.py:90]: Is it supposed to drop this method?

File: openbb_sdk/extensions/charting/openbb_charting/core/plotly.html (hunk 1 above)
Comment (the-praxs) [reply openbb_sdk/extensions/charting/openbb_charting/core/plotly.html:4086]: It had eod somewhere so it got changed. Reverting it.

File: openbb_sdk/extensions/charting/openbb_charting/core/table.html (hunk 2 above)
Comment (the-praxs) [reply openbb_sdk/extensions/charting/openbb_charting/core/table.html:299]: Nope. I thought the charting was having something custom for EOD but this is End of Data so reverting this too.

File: openbb_sdk/extensions/charting/openbb_charting/core/table.html (hunk 3 above)
Comment (the-praxs) [reply openbb_sdk/extensions/charting/openbb_charting/core/table.html:304]: Same as above.

File: openbb_sdk/openbb/package/futures.py (hunk 4 above)
Comment (the-praxs) [reply openbb_sdk/openbb/package/futures.py:90]: Nah. A missing letter on the init caused it. Fixing it.

**Review by hjoaquim — APPROVED** [review]
@the-praxs ftw :rocket:
